h2/src/main/org/h2/api/ErrorCode.java

```diff
@@ -2160,7 +2160,18 @@
      */
     public static final int GENERATED_COLUMN_CANNOT_BE_UPDATABLE_BY_CONSTRAINT_2 = 90155;
 
-    // next is 90156
+    /**
+     * The error with code <code>90156</code> is thrown when function <code>BASE64_ENCODE</code>
+     * or <code>BASE64_DECODE</code> with wrong second parameter <code>algorithm</code> is called
+     *
+     * <pre>
+     * SELECT BASE64_ENCODE(stringtoutf8('my string'),'UNKNOWN');
+     * SELECT BASE64_DECODE('bXkgc3RyaW5n','UNKNOWN');
+     * </pre>
+     */
+    public static final int UNSUPPORTED_ENCODING_ALGORITHM = 90156;
```

Comment:
When you introduce a new error code, you need to add localization, see `h2/src/main/org/h2/res/*`. For this specific function, however, an own error code is not really necessary, you can throw a more generic exception with `throw DbException.getInvalidValueException("Base64 alphabet", value)`.